packs/signoz/templates/volumes/signoz-volume.hcl

```diff
@@ -0,0 +1,12 @@
+namespace = "default"
+name      = "signoz-db"
```

Comment:
As noted on my comment on the README, you have hard-coded names in the volume specs (which is required, unfortunately), but then you have variables for their names in the jobspecs. If you're going to do that, you'll need to have an explanation in the README for submitting the corrected names.